src/Discord.Net.Interactions/InteractionServiceConfig.cs

```diff
@@ -56,6 +59,11 @@ public class InteractionServiceConfig
         ///     Gets or sets delegate to be used by the <see cref="InteractionService"/> when responding to a Rest based interaction.
         /// </summary>
         public RestResponseCallback RestResponseCallback { get; set; } = (ctx, str) => Task.CompletedTask;
+
+        /// <summary>
+        ///     Gets or sets whether an exception should be thrown when a component does not belong to a field in a modal's class.
+        /// </summary>
+        public bool ThrowOnUnknownModalComponent { get; set; } = false;
```

Comment:
oops, seems like i forgot to implement this. ill open a pr for it on your branch.